scripts/extend-expiration-signing-key.sh

```diff
@@ -36,7 +36,7 @@ function extendExpirationSigningKey() {
 	tmpDir=$(mktemp -d -t signing-key-XXXXXXXXXX)
 	local -r gpgDir="$tmpDir/gpg"
 
-		trap '[[ -d "$tmpDir" ]] && rm -r "$tmpDir"' EXIT
+		trap '[[ -d '"$tmpDir"' ]] && rm -r '"$tmpDir"'' EXIT
```

Comment:
I am pretty sure you ran into the issue that $tmpDir was undefined, I wonder why I did not run into it :thinking:
Anyway... we need to improve it as with your current fix we no longer quote the path, potentially running into quoting issues. I'll improve it in a subsequent PR as formatting seems to be off, will improve it as well